origin-dapp/src/pages/profile/VerifyEmail.js

```diff
@@ -168,6 +168,7 @@ class VerifyEmail extends Component {
         </label>
         <div className={`form-control-wrap ${codeErrors ? 'error' : ''}`}>
           <input
+            type="tel"
```

Comment:
It does appear that the email verification code will only consist of number. We should keep this in mind if it changes in the future. 👍